pages/linux/toolbox-create.md

```diff
@@ -0,0 +1,20 @@
+# toolbox create
+
+> Create a new `toolbox` container.
+> More information: <https://manned.org/toolbox-create.1>.
+
+- Create a `toolbox` container for a different distribution other than the host:
```

Comment:
I'm still not fully getting it. So after you created, say, a toolbox for RHEL, you can enter it with `toolbox enter --distro rhel`, right? But what if you run `toolbox enter` without creating the toolbox first? And if `toolbox enter` creates toolboxes on the go, can you use `toolbox rm` to remove it?

(I don't want to be annoying with this, but I think there's still a lot of potential for explaining this command, which in turn helps others to understand it quicker)

Reply:
No issues, I too think toolboxes has a lot of potential for the masses. 
If you perform `toolbox enter` without creating the toolbox first it automatically downloads an image and creates a toolbox. 
For eg, see the reference image below, In fedora, if you enter toolbox enter without creating it will ask you to download an image by default. 
Ref image 1:
![image](https://user-images.githubusercontent.com/26346867/182435202-588441f2-b8a8-4f0e-bff7-9b2cba22076c.png)
Ref image 2:
![image](https://user-images.githubusercontent.com/26346867/182435495-5b76af39-71ba-43e7-a8d8-0dbaee7b4511.png)

What this toolbox container does is it gives you a container environment where you can download and test out packages isolated from your system. Unlike a VM, setting up a toolbox is easier. It is especially useful in immutable operating systems like fedora silver blue, coreos which has a read-only file system allowing only flatpaks (Sandboxed universally packaged applications).